src/Helper/Gravatar.php

```diff
@@ -168,29 +168,66 @@ public function getImgTag()
     {
         $this->setSrcAttribForImg();
         $html = '<img'
-            . $this->htmlAttribs($this->getAttribs())
+            . $this->htmlAttribs($this->getAttributes())
             . $this->getClosingBracket();
 
         return $html;
     }
 
     /**
-     * Set attribs for image tag
+     * Set attributes for image tag
      *
-     * Warning! You shouldn't set src attrib for image tag.
-     * This attrib is overwritten in protected method setSrcAttribForImg().
+     * Warning! You shouldn't set src attribute for image tag.
+     * This attribute is overwritten in protected method setSrcAttribForImg().
      * This method(_setSrcAttribForImg) is called in public method getImgTag().
      *
+     * @param  array $attributes
+     * @return Gravatar
+     *
+     * @deprecated Please use Zend\View\Helper\Gravatar::setAttributes
```

Comment:
We shouldn't have deprecated tag on that method.

Reply:
Whoops